static/src/js/util/accessMethods.js

```diff
@@ -0,0 +1,19 @@
+/**
+ * Determine if the selected access method for a give project collection is valid
+ * @param {Object} projectCollection Project collection config object
+ */
+export const isAccessMethodValid = (projectCollection) => {
```

Comment:
I'm a little confused what a project collection is, is this an object/class or just a payload that matches what used to be a project collection?

Reply:
Its the config out of the store, I'll add that to the comment